src/persist-client/src/impl/state.rs

```diff
@@ -87,12 +87,10 @@ where
     pub fn register_writer(
         &mut self,
         writer_id: &WriterId,
+        heartbeat_timestamp_ms: Option<EpochMillis>,
```

Comment:
why an option? IMO this should just be an `EpochMillis` and aways filled in by the `register_writer` caller